docs/en/latest/pubsub.md

```diff
@@ -42,6 +42,10 @@ In Apache APISIX, the most common scenario is handling north-south traffic from
 
 Currently, Apache APISIX supports WebSocket communication with the client, which can be any application that supports WebSocket, with Protocol Buffer as the serialization mechanism, see the [protocol definition](../../../apisix/pubsub.proto).
```

Comment:
Let's update the path of definition. We should use absolute path as it's not in website

Reply:
changed